raft/src/main/java/org/apache/kafka/raft/RaftConfig.java

```diff
@@ -256,6 +256,21 @@ public static List<Node> voterConnectionsToNodes(Map<Integer, RaftConfig.Address
             .collect(Collectors.toList());
     }
 
+    public static String nodesToVoterConnections(List<Node> nodes) {
```

Comment:
Seems like we're not using this anywhere. Included by mistake perhaps?

Reply:
It was included by mistake.. thanks for pointing out.